Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -434,7 +434,8 @@ private IQueryable<BaseItemEntity> PrepareItemQuery(JellyfinDbContext context, I
         dbQuery = dbQuery.AsSingleQuery()
             .Include(e => e.TrailerTypes)
             .Include(e => e.Provider)
-            .Include(e => e.LockedFields);
+            .Include(e => e.LockedFields)
+            .Include(e => e.UserData);
```

Comment:
Would it be possible to pull out the UserData grouped by UserId? We only ever care about UserData by a specific id so might be worth the optimization

Reply:
Well not entirely, this is also loaded into the LibraryManagers cache where all users load their data from so we need to have all provided. I will see if thats a worthwhile optimisation but from my testing loading everything does not have any noticable performance impact.